### ISSUE TITLE
[Safari] Accent panel opened in a new line cannot be navigated

### ISSUE THREAD

**Issue by f1ames** [ckeditor/ckeditor5-engine#1354]
Extracted from https://github.com/ckeditor/ckeditor5-typing/issues/95.

This issue have been extracted as it is somehow a recurring one. It looks like:
![safari - mar-13-2018 13-22-26](https://user-images.githubusercontent.com/1061942/37341432-a65ed3c8-26c1-11e8-9b87-8029eaf38125.gif)

The last non-working version was `v0.10.0` (however it behaved a little different there so the cause was probably different). When tested in [`v1.0.0-alpha.1`](https://ckeditor5.github.io/releases/v1.0.0-alpha.1/) or [`v1.0.0-alpha.2`](https://ckeditor5.github.io/releases/v1.0.0-alpha.2/) it works fine.

However, when tested on latest `master` (which will probably become a beta release soon) it looks broken (as shown in above gif). Looks like some changes in the meantime fixed it and then broken it again (will take a look with `bisect` later) 🤔 

Anyway, from what I checked, the current cause is manipulating selection during composition (so we are back to #460).

**Comment by f1ames** [comment]
After bisecting - breaking commit:
https://github.com/ckeditor/ckeditor5/commit/e866e4a7bc31ed73fdd0a96f0cf71b5f9bbc3fed -> https://github.com/ckeditor/ckeditor5-font/commit/41e16e0e1c7cf203d8876637b2acc96cb7c2e223

Not sure what happened there, but [previous commit](https://github.com/ckeditor/ckeditor5/commit/dc6364de2e4fd7c767db50d9540a66a8d51d1a58) works fine (or I missed something obvious) 🤔

**Comment by Reinmar** [comment]
The commit you found points to... a ticket in which we changed names of UI components and commands. So, we'd need to be very unlucky :D

**Comment by Reinmar** [comment]
I can reproduce this issue on https://github.com/ckeditor/ckeditor5/commit/dc6364de2e4fd7c767db50d9540a66a8d51d1a58. Did you clear the cache when testing it?

**Comment by Reinmar** [comment]
It WFM on  ckeditor/ckeditor5@764a765 and ckeditor/ckeditor5@12cc47a
It does not work on ckeditor/ckeditor5@222669b.

Which.. is very funny, because it's when we introduced `CompositionObserver`. Perhaps just plugging some listeners to composition events break it?

**Comment by f1ames** [comment]
> The commit you found points to... a ticket in which we changed names of UI components and commands. So, we'd need to be very unlucky :D

Yeah, that was quite confusing :D But still, even when clearing history in Safari before every test (dev console opened), WFM in all three commits you pointed to. And stops working on commit I pointed to...

I do:
```
git checkout #hash
mgit update
npm run test:manual
```

**Comment by f1ames** [comment]
I have recently updated my macOS to `High Sierra` hoping maybe it will change something, but still it works as I described. Maybe this behaviour is influenced by some Safari/system settings.

Anyway, as mentioned in issue description it should be solved by #460.

**Comment by f1ames** [comment]
Native mutations when long-pressing <kbd>a</kbd> in an empty paragraph and then pressing <kbd>Arrow right</kbd> 3 times:

**Chrome**

![image](https://user-images.githubusercontent.com/1061942/40986810-5c9fa07e-68e7-11e8-9a8a-2440caa2a78d.png)

** Safari**

![image](https://user-images.githubusercontent.com/1061942/40986895-90573d0a-68e7-11e8-9ea4-74d4f9cc45d4.png)

A lot more going on in Safari.
